Library/Formula/freeswitch.rb

```diff
@@ -17,6 +17,8 @@ class Freeswitch < Formula
   option "without-sounds-en", "Do not install English (Callie) sounds"
   option "with-sounds-fr", "Install French (June) sounds"
   option "with-sounds-ru", "Install Russian (Elena) sounds"
+  option "without-directory", "Install directory module"
```

Comment:
Nope, but I needed to test whether it would build on travis. I intend to make these opt-in along with some more changes. See: https://github.com/Homebrew/homebrew/issues/45151#issuecomment-149459989